dali/test/python/decoder/test_video.py

```diff
@@ -838,3 +838,82 @@ def decoder_pipeline():
                     + f"{diff_pixels/total_pixels*100}% of pixels. "
                     + f"Expected {ref_frame_bgr} but got {frame_bgr}"
                 )
+
+
+@params(
+    *[
+        (device, codec)
+        for device in ["cpu", "gpu"]
+        for codec in [
+            "h264",
+            "hevc",
+            "vp8",
+            "vp9",
+            "mpeg4",
+        ]  # av1 and mpeg4 are not supported by now
```

Comment:
Unclear comment: mpeg4 is listed above, av1 isn't - so what does it mean?

Reply:
Fixed